From e1ee9756e1c1668641f97a8b8fb53a5f2060c50c Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Wed, 15 Jan 2020 11:49:33 -0800
Subject: [PATCH] Run irqbalance as oneshot by default, without env.

---
 misc/irqbalance.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- irqbalance-1.7.0/misc/irqbalance.service~	2020-08-04 12:30:13.000000000 +0000
+++ irqbalance-1.7.0/misc/irqbalance.service	2020-08-18 14:34:39.734913645 +0000
@@ -7,7 +7,8 @@
 [Service]
 EnvironmentFile=-/usr/lib/irqbalance/defaults.env
 EnvironmentFile=-/path/to/irqbalance.env
-ExecStart=/usr/sbin/irqbalance --foreground $IRQBALANCE_ARGS
+Type=oneshot
+ExecStart=/usr/sbin/irqbalance --oneshot
 CapabilityBoundingSet=
 NoNewPrivileges=yes
 ReadOnlyPaths=/
